src/components/ImageView/index.tsx

```diff
@@ -15,72 +15,59 @@ import useThemeStyles from '@hooks/useThemeStyles';
 import {canUseTouchScreen as canUseTouchScreenUtil} from '@libs/DeviceCapabilities';
 import {isLocalFile} from '@libs/fileDownload/FileUtils';
 import CONST from '@src/CONST';
-import viewRef from '@src/types/utils/viewRef';
 import type ImageViewProps from './types';
 
+type Size = {
```

Comment:
This type is duplicated in `styles/utils/types.ts`

Reply:
I holistically refactored all usages of similar/identical types and created `src/types/utils/Layout.ts`